scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/config/AppConfiguration.java

```diff
@@ -17,7 +17,7 @@
 //TODO: remove backdoor access to directly mounted merlinFileStore (eg via merlin endpoint for downloading mission jars)
 public record AppConfiguration(
     int httpPort,
-    JavalinLoggingState javalinLogging,
+    boolean enableJavalinDevLogging,
```

Comment:
🔶  I think an enum would be more descriptive than a boolean here - would it be possible to revive JavalinLoggingState?

Reply:
Okay, I won't put up to much of a fight on this one 🙂  I'll make my case about enums for future reference.

The nice thing about enums is the values by themselves hold meaning (`JavalinLoggingState.ENABLED` says more than `true` does). Booleans hold no meaning without the name of the function parameter. For example, consider the relative readability of these two lines of code:

```java
new AppConfiguration(JavalinLoggingState.Enabled);
```

```java
new AppConfiguration(true);
```
In the second case, you have to go to the definition of `AppConfiguration` to have any clue what's going on.

The other nice thing is that an enum has room to grow - if we want to add more options, we wouldn't need to change the type of the variables storing it.

Last point, regarding `null`: Our stance on null is that we try to avoid it as much as is practical, preferring to use things like `Optional`, enums, and sealed interfaces instead.

I'll get off my soapbox now and give you an approval 😅